jsx/InstrumentForm.js

```diff
@@ -76,6 +77,15 @@ function renderRadio(element, key, onUpdate, isRequired) {
         hasError={isRequired && (!element.Value)}
         errorMessage="This field is required"
       />
+      <button className="asText" onClick={() => { var el = document.getElementsByName(element.Name);
```

Comment:
This `for` loop isn't needed. Calling `onUpdate` will take care of clearing the radios.
Also I don't think this `span` is doing anything. So you can just do:

```
<button className="asText" onClick={() => { onUpdate(element.Name, null); }} type="button">
  Reset
</button>
```